exercises/the-dice/description.md

```diff
@@ -0,0 +1,11 @@
+Let's play dice! 
+
+The traditional dice is a cube. Each of its six faces shows a different number of dots from 1 to 6. 
+
+Your programming task is related to the analysis of sequences produced by rolling a dice.
+Create a program named Dice to solve the following exercise. The input of the program, a so-called trial, 
```

Comment:
As #321 has it, please write to not have "program". If it helps, you can see how we dealt with this in #528.